package.json

```diff
@@ -16,6 +16,7 @@
     "axios": "^0.21.1",
     "core-js": "^3.6.5",
     "inputmask": "^5.0.5",
+    "moment": "^2.29.1",
```

Comment:
You don't probably need moment.js. Saying more, the developers of moment suggest not to use moment: https://momentjs.com/docs/#/-project-status/

[Day.js](https://github.com/iamkun/dayjs) is pretty the same solution. But I would prefer [date-fns](https://date-fns.org/)

Reply:
Antd Design DatePicker uses moment for props, I didn't want to use it. I will try to replace with date-fns, but I'm not sure will it work or not, not all works with Vue version 3.

Reply:
@Xoxol 
<img width="869" alt="Screen Shot 2021-06-21 at 02 38 08" src="https://user-images.githubusercontent.com/31696608/122691683-e6ad0680-d239-11eb-83fa-39eab8eb4bb6.png">